setup.py

```diff
@@ -127,39 +127,40 @@ def load_requirements(file_list: Optional[Union[str, List[str]]] = None) -> List
         "tsv": tsv_reqs,
         "xlsx": xlsx_reqs,
         # Extra requirements for data connectors
-        "s3": load_requirements("requirements/ingest/s3.in"),
```

Comment:
Don't want to revert all of these.

Reply:
Didn't understand, this is alphabetical reordering. Didn't intend to revert / reorder, is anything wrong in the current state?

Reply:
It's changing the path back: `requirements/ingest/azure.in` -> `requirements/ingest-azure.in`

Reply:
Ah I see, that slipped my eyes

Addressed with: [fix path typos on setup.py](https://github.com/Unstructured-IO/unstructured/pull/1774/commits/025776976a0458dd007734de7ef26b021436b816)